contracts/collections/sg721-base/src/error.rs

```diff
@@ -46,4 +46,13 @@ pub enum ContractError {
 
     #[error("RoyaltyShareIncreased")]
     RoyaltyShareIncreased {},
+
+    #[error("MinterNotFound")]
+    MinterNotFound {},
+
+    #[error("Ownership Update Error: {error}")]
+    OwnershipUpdateError { error: String },
+
+    #[error("found version ({0}) while attempting to migrate from 0.16.0")]
```

Comment:
Hardcoding 0.16.0 here means that a developer has to remember it has to be updated every time.

Reply:
The error I expect is "migration version error" trying to migrate from v1 to v2 for example as commented in `contract.rs`.

Reply:
Going to use this so that error object is not hard coded but not that we are still using the expected from constant: 

```rs
    cw2::assert_contract_version(deps.as_ref().storage, CONTRACT_NAME, EXPECTED_FROM_VERSION)
        .map_err(|_| {
            sg721_base::ContractError::WrongMigrateVersion(
                version.version,
                EXPECTED_FROM_VERSION.to_string(),
            )
        })?;
```